lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -40,6 +40,11 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// Less map declaration
+	if (parent.type === 'atrule' && parent.raws.afterName === ':') {
```

Comment:
I find type errors on the `v14` branch.

```console
$ npm --silent run lint:types
lib/utils/isStandardSyntaxDeclaration.js:45:6 - error TS2532: Object is possibly 'undefined'.

45  if (parent.type === 'atrule' && parent.raws.afterName === ':') {
        ~~~~~~

lib/utils/isStandardSyntaxDeclaration.js:45:34 - error TS2532: Object is possibly 'undefined'.

45  if (parent.type === 'atrule' && parent.raws.afterName === ':') {
                                    ~~~~~~


Found 2 errors.
```

How about adding the following check?

```suggestion
	if (parent && parent.type === 'atrule' && parent.raws.afterName === ':') {
```